PCP_Chapter5_Exercises.cpp

```diff
@@ -30,75 +30,75 @@ double km_ml(double kilometers); // converts kilometers to miles
+    
+    //5-2
+    float radius, volume;
 
-	cout << "\tSphere volume (5-2)\n\n" << "Enter radius(m) of the sphere: ";
-	cin >> radius;
+    cout << "\tSphere volume (5-2)\n\n" << "Enter radius(m) of the sphere: ";
+    cin >> radius;
 
-	volume = sphere_vol(radius);
+    volume = sphere_vol(radius);
 
-	cout << volume << "m^3 is the volume of the sphere \n\n";
+    cout << volume << "m^3 is the volume of the sphere \n\n";
 
-	//5-3
-	int width, height;
+    //5-3
+    int width, height;
 
-	cout << "\tPerimeter of Rectangle (5-3)\n\n" << "Enter its width and height (separated by space): "; // prompts user for both width and height
-	cin >> width >> height; // decided to use cin with several parameters and see functionality.
+    cout << "\tPerimeter of Rectangle (5-3)\n\n" << "Enter its width and height (separated by space): "; // prompts user for both width and height
+    cin >> width >> height; // decided to use cin with several parameters and see functionality.
 
-	cout << "The perimeter of the rectangle is: " << rec_perim(width, height) << "\n\n"; // calculates & prints out resulting perimeter
+    cout << "The perimeter of the rectangle is: " << rec_perim(width, height) << "\n\n"; // calculates & prints out resulting perimeter
 
-	//5-4
-	double kilo;
-	cout << "\tKilometers/h to Miles/h (5-4)\n\n" << "Enter how fast you are going (Km/h): ";
-	cin >> kilo;
+    //5-4
+    double kilo;
+    cout << "\tKilometers/h to Miles/h (5-4)\n\n" << "Enter how fast you are going (Km/h): ";
+    cin >> kilo;
 
-	cout << "you would be going at " << km_ml(kilo) << " Miles/h! \n\n";
-	
+    cout << "you would be going at " << km_ml(kilo) << " Miles/h! \n\n";
+    
 
-	//5-5
-	int time[2];
-	cout << "\tHours and Minutes to just Minutes (5-5)\n\n" << "Enter your hours and minutes (separated by space): ";
-	cin >> time[0] >> time[1];
+    //5-5
+    int time[2];
+    cout << "\tHours and Minutes to just Minutes (5-5)\n\n" << "Enter your hours and minutes (separated by space): ";
+    cin >> time[0] >> time[1];
 
-	cout << "That is " << time[0] * 60 + time[1] << " minutes! \n\n"; // I'm getting good. 
+    cout << "That is " << time[0] * 60 + time[1] << " minutes! \n\n"; // I'm getting good. 
 
 
-	//5-6
-	int hour_min;
-	cout << "\tMinutes to Hour and Minutes (5-6)\n\n" << "Enter your Minutues: ";
-	cin >> hour_min;
+    //5-6
+    int hour_min;
+    cout << "\tMinutes to Hour and Minutes (5-6)\n\n" << "Enter your Minutues: ";
+    cin >> hour_min;
 
-	cout << "That is " << hour_min / 60 << " Hours and " << hour_min % 60 << " Minutes!\n\n"; // Ima beast! 
+    cout << "That is " << hour_min / 60 << " Hours and " << hour_min % 60 << " Minutes!\n\n"; // Ima beast! 
 
 
-	return 0;
+    return 0;
 }
 
-double	cel_fah(double celcius) // body of the function that converts degrees in celcius to farenheit
+double  cel_fah(double celcius) // body of the function that converts degrees in celcius to farenheit
```

Comment:
Start learning to use better names for functions and everything - people reading your code must understand it, and if you make it easier for them the better it is for you and them. A clearer name would have been:

`double fromCelciusToFahrenheit(double celcius);`